### PR TITLE
infrastructure/update-github-templates

### PR DESCRIPTION
**Pull request by hawkticehurst**
**Pull request recommendations:**
- [x] Name your pull request _your-development-type/short-description_. Ex: _feature/read-tiff-files_
- [ ] Link to any relevant issue in the PR description. Ex: _Resolves [gh-12], adds tiff file format support_
- [x] Provide context of changes.

As part of a change across all of CDP's main repositories, I'm updating all of the GitHub issues and PR templates to improve readability and clarity of guidance.

Note: The PR template has been updated to reference content in `CONTRIBUTING.md` that still doesn't exist yet, but will be updated shortly.

- [ ] Provide relevant tests for your feature or bug fix.
- [ ] Provide or update documentation for any feature added by your pull request.

Thanks for contributing!

### CODE REVIEW

**Review by tohuynh — APPROVED** [review]
Looks good to me.

In the PR template, does filling out each header get us "3/3 tasks completed" in the same way that the checkboxes does?

**Comment by hawkticehurst** [comment]
@tohuynh Oh, I'm not sure. 

In fact, I'm not totally sure what you're referring to––could you send a screenshot of what you're talking about?

**Comment by tohuynh** [comment]
https://github.com/CouncilDataProject/cdp-frontend/pulls. On each PR description, theres a 2 of 5 tasks(for each of 5 checkboxes) completed

**Comment by hawkticehurst** [comment]
Ahhh I see it now.

**Comment by tohuynh** [comment]
It's not terribly important though.

**Comment by hawkticehurst** [comment]
Yeah, I'm not sure if it will or not.

I kind of suspect that it will not show up, which means I can add the checkboxes back in. However, I still want to balance that with the usage of titles and spacing because I've found our current PR template pretty hard to read/skim through.

**Comment by tohuynh** [comment]
Yeah, I like the titles and spacing you created too. It's up to you if you can make checkboxes work.

**Comment by hawkticehurst** [comment]
Sweet, I'm leaning towards maybe not having them but I'll let the others chime in too.

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (tohuynh) [inline .github/PULL_REQUEST_TEMPLATE.md:27]: Screenshots are also OK here.

**Review by tohuynh — COMMENTED** [review]
Not sure where we should put this, in the PR template or contributing.md.

If your component has presentational text (non-data text), you should add the text to `src/assets/strings` and provide translation for currently supported languages. Then use `strings.<text>` in your component. This is to be consistent with our text localization feature. 

@BrianL3 might have more thoughts on this.

**Review by JacksonMaxfield — APPROVED** [review]
Approving because I think these could be merged now but also agree with @tohuynh that maybe adding a comment about "string / localization management" somewhere in the PR template.

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (JacksonMaxfield) [reply .github/PULL_REQUEST_TEMPLATE.md:27]: I slightly disagree personally. I want to see the component work / load on _any_ browser rather than just a screenshot personally. :shrug: 

That said, it's minor and I wouldn't care too much if someone dropped screenshots.

Comment (tohuynh) [reply .github/PULL_REQUEST_TEMPLATE.md:27]: Yeah, screenshots for minor changes are acceptable. I like to preview the changes on any browser too.
But in the case where a contributor has multiple PRs open, they might not be able to have a gh-page deployment for each PR.

Comment (JacksonMaxfield) [reply .github/PULL_REQUEST_TEMPLATE.md:27]: Very fair point on that one.

Comment (tohuynh) [reply .github/PULL_REQUEST_TEMPLATE.md:27]: Overall, provide a link to gh-page if you can. If not, then provide screenshots.

**Comment by hawkticehurst** [comment]
Sounds good, I think I'll hold off on adding that for now until we create a section in CONTRIBUTING.md that discusses the localization.

**Pull request merged**
